Zend/tests/first_class_callable_003.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+First Class Callable from Method
+--FILE--
+<?php
+class Foo {
+    public function bar()  {
+        echo "OK";
+    }
+}
+
+$foo = new Foo;
+$bar = $foo->bar(...);
```

Comment:
what about `$a = 'foo'; $foo->{$a}(...);`? named params?

Reply:
The whole point of this PR is the implementation of what is described in [this RFC](https://wiki.php.net/rfc/first_class_callable_syntax), in short: the possibility to avoid function or method's name being stored in strings/arrays.
This line makes use of that new feature in a file that has the purpose of **testing** it.

Reply:
yes, and it must work with dynamic function name